.github/scripts/simple-smoke-test.sh

```diff
@@ -6,12 +6,27 @@ set -o pipefail
 function test_docker_image() {
     local image=$1
     local container_name=$2
+    local expected_distribution_type=$3
 
     if [ "$(docker ps --all --quiet --filter name="$container_name")" ]; then
       echo "Removing existing '$container_name' container"
       docker container rm --force "$container_name"
     fi
 
+    echo "Checking if $image is EE"
```

Comment:
[nit] could be simplify (not sure about correct syntax)

```
[[ docker run --rm $image bash -c 'compgen -G lib/*enterprise* ]] && distribution_type="ee" || distribution_type="oss"
 echo "Image type $distribution_type"

```

Reply:
I don't think that's functionally equivalent - it doesn't log either way.